modules/table/src/main/java/org/apache/ignite/internal/table/distributed/replicator/PartitionReplicaListener.java

```diff
@@ -1740,8 +1740,10 @@ private CompletableFuture<ReplicaResult> processMultiEntryAction(ReadWriteMultiR
 
                         if (lockedRowId != null) {
                             rowIdsToDelete.put(lockedRowId.uuid(), null);
+
+                            result.add(new NullBinaryRow());
```

Comment:
Let's create and add a ticket to replace it with the `BitSet`.

Reply:
Added TODO comment about it.